frontend/src/components/Navbar/DropDownMenu.tsx

```diff
@@ -0,0 +1,60 @@
+import './DropDownMenu.css';
+
+import { ReactComponent as SavedIcon } from 'assets/icons/saved.svg';
+import { ReactComponent as SettingsIcon } from 'assets/icons/settings.svg';
+import { ReactComponent as ProfileIcon } from 'assets/icons/profile.svg';
+import { ReactComponent as SwitchAccountIcon } from 'assets/icons/switch-accounts.svg';
+import { ReactComponent as ReportProblemIcon } from 'assets/icons/report-problem.svg';
+
+export default function DropDownMenu() {
```

Comment:
`DropDownMenu` is a very generic name for this component. This is not a reusable drop-down component.

Give it a more appropriate name.